packages/service-worker-mock/NOTICE

```diff
@@ -0,0 +1,2 @@
+Parts of sources of this package are derivates from https://github.com/GoogleChrome/workbox,
+licensed under MIT, Copyright 2018 Google LLC
```

Comment:
We should remove all the mentions to Google licensing. I don't want to fight with the Pinterest lawyers. MIT allows for copy/paste freely, so just a notice that these are derived from the workbox project is enough without mentioning licensing.

Reply:
✅